### ISSUE TITLE
About training Mask R-CNN

### ISSUE THREAD

**Issue by YoruCathy** [qqlu/Amodal-Instance-Segmentation-through-KINS-Dataset#12]
Hello, you listed result on Mask R-CNN, with det result around 31. I tried to train Mask RCNN on KINS dataset but I just can't get that score.
Pleazzzzzzzz tell me how did you train Mask R-CNN? How many GPUs did you use, and your batch size, lr, lrdecay and some other parameters?
Thanks a lottt

**Comment by qqlu** [comment]
Hi,  8 GPU was used with batch size 16 and min size 1666. I think your image size may be very small.

**Comment by YoruCathy** [comment]
Ah, my images are resized to 1333*800 keeping ratio, maybe I have to make them larger
By the way how many epochs or iterations did you train?

**Comment by qqlu** [comment]
The hyper-parameters are set in my reference code (configs/kitti/). The max iterations are 24000
 with steps [0, 20000, 22000].

**Comment by YoruCathy** [comment]
Oh I see, thank you (●'◡'●)
